src/main/resources/api/service_openapi.yaml

```diff
@@ -247,9 +247,7 @@ paths:
       responses:
         '204':
           description: Role granted successfully
-        '403':
-          $ref: '#/components/responses/PermissionDenied'
-        '404':
+        '401':
           $ref: '#/components/responses/NotFound'
```

Comment:
It seems like 401 should give a different response type than 404. I get why you avoid doing 404 though.

If not found is the predominant situation, then why not return 404 in all cases? If that's more likely to be clear.

Reply:
Yeah, this is one I'm not sure about. We intentionally give the same response to stuff that should be a 401 vs. 404, and we expect to see both cases. It's hard to say one is predominant here.

I picked the `NotFound` object here because its description explicitly calls out "Not found (or unauthorized)". The only difference between the response types here is the description string that gets used in swagger-ui - otherwise, they're both just ErrorReport objects.